modules/programs/vscode.nix

```diff
@@ -79,7 +79,7 @@ in
       let
         toPaths = path:
           let
-            p = "${path}/share/vscode/extensions";
+            p = "${path}";
```

Comment:
Since the string now is so short you could inline the `p` variable. I.e. to remove the `let` and change the body to.
```
  "${extensionPath}/${k}".source = "${path}/${k}";
}) (builtins.readDir path);
```